types/good-logs/README.md

```diff
@@ -0,0 +1,206 @@
+
```

Comment:
This readme will not be included in the package published to npm; I would remove this file.

Reply:
Yes please, thank you.